circle.yml

```diff
@@ -0,0 +1,8 @@
+machine:
+  ruby:
+    version: 2.2.1
```

Comment:
So we now have stories in our backlog to do the upgrade for all apps and gems. But it'll be handled as a separated piece of work.